physcraper/__init__.py

```diff
@@ -1981,6 +2017,39 @@ def read_local_blast_query(self, fn_path):
                                                query_dict[key]["evalue"]))
                 fn.close()
 
+    def get_taxid_from_acc(self, gb_acc):
```

Comment:
This is only for local blast, right? Maybe put in name, and or add an assert right at the beginning of the function that the configuration is for local blast.

Reply:
see above